crates/configuration/src/network.rs

```diff
@@ -0,0 +1,92 @@
+use crate::{
+    shared::types::{IpAddress, MultiAddress, Timeout},
+    HrmpChannelConfig, ParachainConfig, RelaychainConfig,
+};
+
+/// Global settings applied to an entire network.
+pub struct GlobalSettings {
+    /// Whether we should spawn a dedicated bootnode for each chain.
+    spawn_bootnode: bool,
+
+    /// External bootnode address.
+    /// [TODO]: is it a default overriden by node config, maybe an option ?
+    bootnode_address: MultiAddress,
```

Comment:
Yes, the expected behavior is that this bootnode is added to the `relaychain` spec. But we can drop this field and revisit later. We don't have any use-case that use this feature.

```suggestion
    /// External bootnode address.
    /// [TODO]: is it a default overriden by node config, maybe an option ?
    bootnode_address: MultiAddress,
```